data/tutorials/gs_00_up_and_running.md

```diff
@@ -11,88 +11,166 @@ date: 2021-05-27T21:07:30-00:00
-$ which ocaml
-/Users/frank/.opam/4.14.0/bin/ocaml
+The second command (`eval $(opam env)`) modifies a few environments variables to make the shell aware of the switch you are using. For instance, it will add what is needed to the `PATH` variable so that typing `ocaml` in the shell runs the ocaml binary of the current switch.
+
+In case you are not satisfied with the `ocaml` version of your system switch, you can write the following commands to create a new switch with a recent version of ocaml:
 
-$ ocaml -version
-The OCaml toplevel, version 4.14.0
+``` shell
+opam switch create 4.14.0
+eval $(opam env)
 ```
 
-As an alternative **for either Linux or macOS**, a binary distribution of Opam is
-available:
+More information can be found on the [official website](https://opam.ocaml.org/).
+
+### The OCaml base tools
+
+OCaml is installed in an Opam switch, which, among others, bring the following
+programs:
+
+- A "toplevel", which can be called with the `ocaml` command. It consists of a read-eval-print loop (a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)), similar to the `python` or `node` command, and can be handy to quickly try the language. The user interface of `ocaml` is very basic but is improved a lot in one of the package that we will install later: `utop`.
+
+- A compiler to **native code**, called `ocamlopt`. It creates executables that can be executed directly on your system.
 
+- A compiler to **bytecode**, called `ocamlc`. It creates executables that can be interpreted by a variety of runtime environments, making it more flexible.
+
+Although it is theoretically all we need to write OCaml code, it is not at all a complete and comfortable development environment.
+
+## Setting up development tools
+
+This step is done automatically by the alpha-version platform installer, [see below](#up-and-running-with-the-platform-installer).
+
+We will now install everything we need to get a complete development environment, which includes:
+
+- `dune`, a fast and full-featured build system for OCaml,
+- `merlin` (the backend) and `ocaml-lsp-server` to provide editors with many useful features such as "jump to definition",
+- `odoc` to generate documentation from OCaml code,
+- `ocamlformat` to automatically format OCaml code,
+- `utop`, an improved REPL,
+- `dune-release` to release code to `opam-repository`, the package base for `opam`.
+
+All these tools can be installed in your current switch (remember that `opam` groups installed packages in independant switches) using the following command:
+
+``` shell
+opam install dune merlin ocaml-lsp-server odoc ocamlformat utop dune-release
 ```
-sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)
+
+Now that the tools are installed, it remains to understand how to use them. Most of them will be driven either by the editor or by `dune`, but `utop` is handy to try OCaml or a specific library.
+
+## Up and Running with the platform installer
```

Comment:
I like that we keep the two workflows in the documentation, but I would invert the two: at the moment getting Up and Running with the installer is the alternative, I would make Platform installer workflow the default.

I would also remove the disclaimers that it's an early version and still in development: the PR won't be merged before the first stable version anyways, at which point, this will become the recommended way to install OCaml and the Platform.